packages/layout/src/components/cards/components/button.tsx

```diff
@@ -33,28 +34,70 @@ export const UnderlinedButton: React.FC<UnderlinedButtonProps> = ({
 	tabIndex,
 	isEditButton,
 	onCollapseCallback,
-	btnRef,
+	buttonRef,
 	isMainHeading = false,
+	notHeading = false,
 }) => {
-	if (typeof onClick === 'undefined') {
-		return (
-			<div className={styles.buttonPlaceholder}>
-				<Flex cfg={{ flex: '0 0 auto', alignItems: 'center' }}>
-					{isMainHeading ? (
-						<H3 cfg={{ fontWeight: 3 }}>{children}</H3>
-					) : (
-						<H4>{children}</H4>
-					)}
-				</Flex>
-			</div>
-		);
-	}
+	const editViewButtonRef = useRef(null);
+
+	const NotClickableButton: React.FC = () => (
+		<div className={styles.buttonPlaceholder}>
+			<Flex cfg={{ flex: '0 0 auto', alignItems: 'center' }}>
+				{isMainHeading ? (
+					<H3
+						cfg={{ fontWeight: 3, fontSize: 3 }}
```

Comment:
Does this not pick up global h3 styling? If not can the existing `.h3` class be applied? If not, create a class rather than repeating styling in multiple places. (Same comment applies to h4 and possibly the paragraph too.)